qa/L0_metrics/metrics_config_test.py

```diff
@@ -97,6 +98,17 @@ def test_cache_counters_missing(self):
         for metric in CACHE_COUNTER_PATTERNS:
             self.assertNotIn(metric, metrics)
 
+    # Histograms
+    def test_inf_histograms_decoupled_exist(self):
+        metrics = self._get_metrics()
+        for metric in INF_HISTOGRAM_DECOUPLED_PATTERNS:
+            self.assertIn(metric, metrics)
```

Comment:
Since we saw earlier tonight that a recent change might've broken the metric and the buckets weren't actually getting updated - please add a test that verifies the buckets get updated after an inference so we can be confident it is still working when making changes